### DISCUSSION TITLE
命令提示字元操作

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#248] [General]
在撰寫程式，如想將github的儲存庫藉由git clone的方式下載一份到電腦(本機端，本地端，local)上，如果電腦是windows作業系統(OS)，就一定要使用命令提示字元(command line, cmd)來完成。我在這邊把會用到的cmd指令整理如下。

1. `dir`
dir就是directory(目錄)，而「目錄」就是「資料夾」。執行dir指令可以看到處於當前路徑資料夾下(下一層，不是同一層)的所有資料夾與檔案。
2. `cd 資料夾名稱(folder name)`
`cd 資料夾名稱(folder name)`可以找到(存取到，access)當前路徑的資料夾的下一層的某一個資料夾或檔案。
3. `cd ../`
回到路徑上一層（前一層）

**Comment by enoch-lightning** [comment]
`code .`的.是根目錄，code就是visual studio code。`code .`就是在根目錄開啟visual studio code，也可以說是用visual studio code開啟根目錄。

**Comment by lx3211** [comment]
如果系統是macOS的話，與dir相對應的指令就是ls(list)

**Comment by 32844583** [comment]
順帶一提 cd ../.. 就可以回到上上一層囉 !

**Comment by gnsJhenJie** [comment]
分享一個小訣竅，當打了一大串指令(像是`cd /home/user/Desktop/abc/12345`)後，游標會停在最尾端，這時如果想更改前面的文字，大部分的人會按鍵盤方向鍵一直讓游標往右，但這在指令很長時實在很沒效率，這時可以直接`Ctrl + A`讓游標指向最前端，而`Ctrl + E`則能讓游標指向最尾端。

**Comment by NcuJerryKao** [comment]
當看到 命令提示字元 我嚇得半死
但一看你的內容 竟然發笑了

**Comment by BingRong1217** [comment]
很有用也很基本的知識！

**Comment by xCk27x** [comment]
哈哈 也來補充個
開啟cmd的快捷鍵是「windows+R」
視窗開啟後按Enter 預設是打開cmd
用習慣後就覺得很方便了~
不用滑鼠幫忙就能開啟cmd

**Comment by limitx0** [comment]
在Windows上如果用的是PowerShell的話dir和ls都是一樣的功能
